site/content/zh-CN/docs/reference/kubectl-kueue/commands/kueuectl_version/_index.md

```diff
@@ -208,7 +202,7 @@ kueuectl version
     <tr>
         <td></td>
         <td style="line-height: 130%; word-wrap: break-word;">
-            <p>Bearer token for authentication to the API server</p>
+            <p>用于 API 服务器身份验证的 Bearer 令牌</p>
```

Comment:
```suggestion
            <p>用于 API 服务器身份验证的持有者令牌</p>
```
Refer to:
https://kubernetes.io/zh-cn/docs/contribute/localization_zh/#glossary